app/models/service_template.rb

```diff
@@ -95,7 +95,7 @@ def self.all_catalog_item_types
         "generic_orchestration" => N_("Orchestration"),
       }
 
-      ExtManagementSystem.supported_types_for_catalog
+      ExtManagementSystem.supporting(:catalog)
```

Comment:
Ah no I see why the tests are failing, this gets back to what I was talking about re supported classes vs supported instances.

The previous `ExtManagementSystem.supported_types_for_catalog` returns supported classes, `      ExtManagementSystem.supporting(:catalog)` returns instances from supported classes so this now returns `[]` where the previous method returned a number of provider classes

Reply:
Good eye. looks like we want `subclasses_supporting` rather than instances of ems that support.
Subtle.